cime_config/cesm/machines/config_compilers.xml

```diff
@@ -69,7 +69,7 @@ using a fortran linker.
      compilers -->
 <compiler>
   <CPPDEFS>
-    <append> -D<env>OS</env> -DCPR<shell> echo <env>COMPILER</env> | tr a-z A-z</shell></append>
```

Comment:
@mfdeakin-sandia Please note this change - the shell command was not working in cmake.